.github/workflows/ci.yml

```diff
@@ -29,10 +29,6 @@ jobs:
         os: [ubuntu-latest, macos-latest]
         swift: ["5.10"]
         include:
-        - os: macos-12
-          swift: "5.10"
-        - os: ubuntu-22.04
-          swift: "5.10"
```

Comment:
These lines were added in #473, when `macos-12` and `ubuntu-22.04` were the latest versions. If that was the intent, the lines are unnecessary because the matrix already tests for `ubuntu-latest` and `macos-latest`.

Reply:
Yeah I agree 🙂